@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -691,12 +691,32 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     wrappingMode?: DynamicProperty<"None" | "Character" | "Word">;
     /**
      * Text position regarding the baseline.
+     *
+     * @deprecated Use [[hPlacement]] attribute, which defines label text placement in exactly
+     * opposite way, text that were `Left` aligned are by deduction `Right` placed relative to
+     * the label icon.
      */
     hAlignment?: DynamicProperty<"Left" | "Center" | "Right">;
     /**
      * Text position inside a line.
+     *
+     * @deprecated Use [[vPlacement]] attribute instead.
      */
     vAlignment?: DynamicProperty<"Above" | "Center" | "Below">;
+    /**
+     * Text position regarding the baseline.
+     *
+     * @deprecated Use [[hPlacement]] attribute, which defines label text placement in exactly
+     * opposite way, text that were `Left` aligned are by deduction `Right` placed relative to
+     * the label icon.
+     */
+    hPlacement?: DynamicProperty<"Left" | "Center" | "Right">;
+    /**
+     * Text position inside a line.
+     *
+     * @deprecated Use [[vPlacement]] attribute instead.
+     */
+    vPlacement?: DynamicProperty<"Top" | "Center" | "Bottom">;
```

Comment:
what about TextTechniqueParams? We have to do the same change there.

Reply:
I did not added it there, because as far as I know the change (placement) regards to the point based labels. I don't exactly know what the meaning would be for simple text... could you clarify.

Reply:
For text it would be the placement relative to the text anchor.
As we've seen before in native the have "placement" attribute for all labels.